Adding events and partnerships page (#10762)
PostHog/posthog.com: contents/handbook/growth/marketing/events-partnerships.mdx

---
title: Partnerships and Events
sidebar: Handbook
showTitle: true
---

> At PostHog, when we hear "events," we tend to think of the actual event data sent to PostHog for analytics. Sorry-not-sorry.
> When it comes to community-based, real-world or virtual gatherings, PostHog is newer to these kind of experiences. 

Reviewer: ```suggestion
This page covers how we organize real-world or virtual events for our community, not [internal events](/handbook/company/offsites) or [app events](/docs/data/events). 
```

We have so far:
* Sponsored the [ElevenLabs + x16z Worldwide Hackathon](https://hackathon.elevenlabs.io/)
* Hosted our own developer meetup in London

> * Hosted several of our own [company offsites](handbook/company/offsites)

Reviewer: Offsite =/= events, tbh

However, we're definitely interested in developing a uniquely PostHog approach to events. If you're a community partner with an idea for a quirky, developer-centric, interesting event, please reach out to [Kevan](/community/profiles/32499).

Here is what we are learning from our first steps into events:

> ## Sponsoring an event:

Reviewer: ```suggestion
## Sponsoring an event
```

> ### How to filter a potential event sponsorship opportunities:

Reviewer: ```suggestion
### How to filter a potential event sponsorship opportunities
```

- **Is the host brand interesting?** We want the hosts themselves to be brands we’re keen on. That should mean they’re serving our ICP and industry, and ideally doing their own interesting, innovative thing in the world.
- **Is the event itself interesting?** Check the event’s purpose, principles, and participants before agreeing. We don’t do boring partnerships or events. We focus on makers. We like new things. 
- **Can PostHog help bring it to the next level?**  Ideally, our role is to make an event “more interesting,” not more of the same. This can include practical technology components and demos, an interesting branded experience or collab.

> - **Is there a reasonable time window for collaborating?** Aim for a minimum of 8 weeks out. We find that 4 weeks out is the latest we can get involved in an event, because of the need to arrange physical goods, travel, attendance and promotion. 

Reviewer: ```suggestion
```
I don't think we need to be prescriptive here. We've definitely moved faster than this on several occasions and 8 weeks is an eon in PostHog time. "Make sure you have time to do the things you want to do" is also advise we should take as read.

Author: Hmmm...shipping times to North America for merch are definitely a barrier. Booking international travel is definitely a factor, especially for people with families. I think I would not delete this, but could make it less prescriptive/specific on time-frames. 

(Sidenote: "Take it as read" seems to be a British-ism. I looked it up, as I'm not familiar with the phrase. Is it basically "we should assume everybody knows this already and it doesn't need to be said?" And: do you pronounce it "red" or "reed"?)

> ### Checklist of questions for sponsoring events: 

Reviewer: ```suggestion
### Checklist of questions for sponsoring events
```

Ask these questions to determine what specific tasks PostHog might need to do: 
- **People:** Are PostHog people needed in person? What level, role or skill? 
- **Asks of the people:** Will there be a talk? Networking? Demos? Participation?
- **Goods/merch:** Will physical objects (shirts, giveaways, signage) be needed?
- **Social media:** What kind of co-promoting is expected?
- **Artwork and branding:** Are unique, custom assets needed?
- **Online support:** Will live and realtime support or communication be needed during the event?
- **Communication:** In-house at PostHog, and beyond, what needs to be shared? 
- **Travel:** Is travel going to be part of it? Who, when, where, how? 

> - **Timeline:** When is it happening, and when do we need all this by?

Author: That's a cool idea; thank you!

> ### Make your plan and build your team:

Reviewer: ```suggestion
### Make your plan and build your team
```

> - Based on the above checklist, pick an event owner, and if needed, a temporary small team. This team dissolves after the event, but takes seriously the work of delivering well. 

Reviewer: ```suggestion
- Pick an event owner who is responsible for coordinating the event and making sure that everything goes smoothly. 
```

- Create an Issue that covers the event sponsorship, and assign it to the event owner. [Example event Issue](https://github.com/PostHog/company-internal/issues/1759)

> ### Showing up to a partner event sponsorship as PostHog:

Reviewer: ```suggestion
### Participating in partner events
```

Don't need `:` in a H1, H2 or H3

- Travel in such a way that you can arrive on-time, well-rested, early and ready to support well
- Plan in advance to bring everything you need: laptop, adapters, slides, merch, signs, etc.
- Have speaking remarks prepared, and anticipate playing a role speaking on-stage to give PostHog further visibility 
- Wear PostHog merch so you are visible part of the team
- Act as a co-host, supporting the hosting team, and offering encouragement as the event goes along
- While business cards are perhaps antiquated, have a plan for sharing contact info with people you meet (LinkedIn, Bump, etc)
- Be on a mission to meet people; it’s part of the event function. Ask and answer questions. 
- Don’t cluster in a clique: If attending with our PostHog teammates, spread out, don’t loiter as a wolfpack
- Finish line: know what constitutes “done” for your presence at the event (don’t leave early or without checking with the host)

> ## Planning and running a PostHog branded event:

Reviewer: We can probably just cut this entire section. We have no plans to run our own events regularly and the events we do occasionally run (e.g. AI startup school idea) are so bespoke that the guidance here won't be all that useful.

> ### Answer these questions before you decide to do an event.

Reviewer: ```suggestion
### Answer these questions before you decide to do an event
```

- **Harvest:** Plan for what you need to take away from the event (prototypes, contact info, decisions, notes)
- **After:** What needs to continue after the event? (Action, data clean-up, relational care, debrief)

### Invitations and promotion 
“An event starts the moment its participants hear about it.”
* Key team and VIP attendees should get personalized invites (ideally, a realtime conversation)
* Ideal participants should get direct outreach (find them where they are)
* Advertising publicly (social media, ads) is the third wave once you’ve secured your core. Ideally, your event is more than 50% subscribed before it’s publicized.

### Securing a venue & food
- Consider space design (lighting, vibe, mood) as it applies to your purpose and principles.
- Knowing what works for your core participants matters. Consider commute times, parking, transit, access.
- Cost is a big factor, but often known partners with physical spaces can strike deals.
- Catering is as central as the venue. Pick a venue compatible with your food plans.
- Plans for generous, available coffee and snacks, and if supplying meals, be conscious of attendee’s dietary needs. Ask during registration.

> ### Plan for other physical elements:

Reviewer: ```suggestion
### Plan for other physical elements
```

* Merch, lanyards, name tags, signage, handouts and other display materials need to be considered early

> ### Before the event:

Reviewer: ```suggestion
### Before the event
```

- **Participant communication:** 
    - Reduce the need for multiple third-party tools (partiful, discord, etc). Manage RSVPs, attendee communication, instructions, with as few possible steps, reduced logins, ideally with one system.
    - Ensure instructions are retrievable (ie, website, google doc or app), not ephemeral (dms, email)
- **Sponsor communication:** Create a clear sponsor kit with specific asks. That should include needs and expectations around:
    - Physical attendance: Are people needed in person? Will there be a talk? Networking? Demos? Participation?
    - Goods/merch: Will physical objects (shirts, giveaways, signage) be needed?
    - Online support: Will live and realtime support or communication be needed during the event?
    - Social media: What kind of co-promoting is expected?
    - Artwork and branding: Are unique, custom assets needed?
- **Technology testing and prep:**
    - If involving live building or hacking, do a test run to ensure the products perform as expected
    - If integrating with third parties, test those integrations 

> ### At the event space: 

Reviewer: ```suggestion
### At the event space
```

- Arriving at a physical event space: Ensure clear signage for wayfinding. 
- Staffing: Ensure event staff are present and available:  greet and welcome people, confirm registration.
- For hosting and welcoming:  Ensure host has experience public speaking.
- When involving sponsors and making their tech part of the experience, each of those tech products should be tested, have live support available
- Instructions for attendees need to be centralized, repeated, clearly accessible 

> ### During the event:

Reviewer: ```suggestion
### During the event
```

---
title: Hosting and sponsoring events
sidebar: Handbook
showTitle: true
---

This page covers how we organize real-world or virtual events for our community, not [internal events](/handbook/company/offsites) or [app events](/docs/data/events). 

We have so far:
* Sponsored several hackathons and events, such as the <PrivateLink url="https://github.com/PostHog/company-internal/issues/1759">ElevenLabs + x16z Worldwide Hackathon</PrivateLink>.
* Hosted <PrivateLink url="https://github.com/PostHog/company-internal/issues/608">our own developer meetup in London</PrivateLink>

However, we're definitely interested in developing a uniquely PostHog approach to events. If you're a community partner with an idea for a quirky, developer-centric, interesting event, please reach out to [Kevan](/community/profiles/32499).

Here is what we are learning from our first steps into events:

## Sponsoring an event

> 

Reviewer: ```suggestion
Our goal with all our events is to connect with our community in a genuine, human way. We don't just sponsor events where we get our logo on the brochure for 'brand exposure' and nothing else. 

```

> ### How to filter a potential event sponsorship opportunities

Reviewer: ```suggestion
### How to filter potential event sponsorship opportunities
```
